README.md

```diff
@@ -42,7 +42,7 @@ As of now, these are the supported resources:
 
 ### Prerequisites
 
- - Make sure you have `JDK 8` installed.
+ - Make sure you have `JDK 11` installed.
```

Comment:
Why is it required if docker container takes care of it?

Reply:
I thought it is required to install jdk all the time.
If so, this pre-requisites is no longer applicable.

Reply:
@butenkor I have created another PR for this document change.
https://github.com/commercetools/commercetools-project-sync/pull/259